src/rules/font-weight-notation/index.js

```diff
@@ -46,11 +48,20 @@ export default function (expectation, options) {
     })
 
     function checkFont(decl) {
+      const valueList = postcss.list.space(decl.value)
+      // We do not need to more carefully distinguish font-weight
+      // numbers from unitless line-heights because line-heights in
+      // `font` values need to be part of a font-size/line-height pair
+      const hasNumericFontWeight = valueList.some(isNumbery)
+      console.log(JSON.stringify(hasNumericFontWeight))
+
       for (let value of postcss.list.space(decl.value)) {
-        // We do not need to more carefully distinguish font-weight
-        // numbers from unitless line-heights because line-heights in
-        // `font` values need to be part of a font-size/line-height pair
-        if (isNumbery(value) || NAMED_WEIGHTS.indexOf(value) !== -1) {
+        console.log(JSON.stringify(value))
```

Comment:
and here. Once that up stream eslint issue is resolved, we'll get `no-console` going here.